components/html/modules/content-patterns/_typography.html

```diff
@@ -3,79 +3,79 @@
 				<aside class="highlight">
-					<h3 class="highlight__title">Archivo Bold 18px Caps #000000</h3>
+					<h3 class="highlight__title">Uses the H5 Style</h3>
 					<p class="highlight__content">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aspernatur ipsum perspiciatis veniam, rem totam optio distinctio. Porro, nisi in unde, nemo nulla totam corporis beatae natus, tempora hic aliquid. Voluptatum.</p>
 				</aside>
-				<h5>H5 Archivo Bold 22px Caps #000000</h5>
+				<h5>H5 Archivo Narrow Bold 1.375em Caps #000000</h5>
 				<p>Explicabo, impedit repudiandae maxime. Commodi, quod earum voluptatem dolorem aliquam omnis repellendus vel quisquam similique dolores, esse maxime.</p>
-				<h6>H6 Archivo Bold 18px Caps #000000</h6>
+				<h6>H6 Archivo Narrow Bold 1.125em Caps #000000</h6>
 				<p>Aliquam officia aliquid est earum minus qui nulla. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Obcaecati aliquid, aspernatur exercitationem. Commodi, quod earum voluptatem dolorem aliquam omnis repellendus vel quisquam similique dolores, esse maxime.</p>
 				<aside class="quote">
-					<h3 class="quote__content">"Archivo Narrow Bold 24px #2e2e2e Consectetur adipisicing elit. At illum esse quas sunt, reiciendis magnam itaque aut exercitationem et, voluptatum laboriosam expedita, quia officia ipsum nemo molestias alias sequi vero."</h3>
-					<p class="quote__source">- Archivo Narrow Bold 18px #98700d</p>
+					<h3 class="quote__content">"Archivo Narrow Bold 1.5em #2e2e2e Consectetur adipisicing elit. At illum esse quas sunt, reiciendis magnam itaque aut exercitationem et, voluptatum laboriosam expedita, quia officia ipsum nemo molestias alias sequi vero."</h3>
+					<p class="quote__source">- Archivo Narrow Bold 1.125em #98700d</p>
 				</aside>
 				<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sed quidem similique, aspernatur illum tempora, nostrum pariatur eos consequuntur atque minus fugit deleniti illo reprehenderit architecto quis ipsam debitis asperiores temporibus!</p>
-				<aside class="quote quote-quotemark">
-					<h3 class="quote__content">"Archivo Narrow Bold 24px #2e2e2e Consectetur adipisicing elit. At illum esse quas sunt, reiciendis magnam itaque aut exercitationem et, voluptatum laboriosam expedita, quia officia ipsum nemo molestias alias sequi vero."</h3>
-					<span class="quote__source">- Archivo Narrow Bold 18px #98700d</span>
+				<aside class="quote quote-mark">
+					<h3 class="quote__content">"Archivo Narrow Bold 1.5em #2e2e2e Consectetur adipisicing elit. At illum esse quas sunt, reiciendis magnam itaque aut exercitationem et, voluptatum laboriosam expedita, quia officia ipsum nemo molestias alias sequi vero."</h3>
```

Comment:
Although this is just an example, we probably should use different element for the quote content, such as a `<p>`.

Reply:
Good catch. I think that just got copied and never changed.